quint/testFixture/_1006unexpectedExpr.json

```diff
@@ -1 +1 @@
-{"stage":"parsing","warnings":[],"errors":[{"explanation":"Failed to resolve name name in definition for access, in module unexpectedExpr","locs":[{"source":"mocked_path/testFixture/_1006unexpectedExpr.qnt","start":{"line":2,"col":21,"index":84},"end":{"line":2,"col":24,"index":87}}]}]}
\ No newline at end of file
+{"stage":"parsing","warnings":[],"errors":[{"explanation":"[QNT404] Name 'name' not found","locs":[{"source":"mocked_path/testFixture/_1006unexpectedExpr.qnt","start":{"line":2,"col":21,"index":84},"end":{"line":2,"col":24,"index":87}}]}]}
```

Comment:
Shall we automatically exclude these json files from reviews, similar to how we have done it in the tutorials, see `tutorials/.gitattributes`

Reply:
I like having them in the diff so the behavior changes are somewhat explicit in the review, i.e. here you can see a concrete error message diff. Of course, when it's just ids moving around, it's noise. But most importantly, this is a sanity check that we didn't just automatically update the fixtures to error outputs and our unit tests are passing only because the expectation is for things to error out.

Reply:
Oh damn, that actually breaks integration with Apalache because this is part of our JSON output. I'll roll back the change to get this merged and we can change the name in a new PR   in sync with an Apalache release.

Reply:
(wrong thread, this comment was about the id vs reference)